changelogs/fragments/odbc.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- "odbc - Added a parameter 'commit' which allows users to subvery the explicit commit after the execute call"
```

```suggestion
- "odbc - added a parameter ``commit`` which allows users to subvery the explicit commit after the execute call (https://github.com/ansible-collections/community.general/pull/1139)."
```
I think "subvery" should be replaced by another word? Maybe "disable"?

---

Or did you mean "subvert"? I guess disable is easier to understand for non-native speakers though.